.pre-commit-config.yaml

```diff
@@ -26,3 +26,12 @@ repos:
     rev: 6.1.0
     hooks:
     -   id: flake8
+-   repo: https://github.com/codespell-project/codespell
+    rev: v2.1.0
+    hooks:
+    - id: codespell
+      name: 'Spell checker'
+      entry: codespell
+      language: python
+      exclude: '^frontend/(build|node_modules)/|.*\.lock$'
```

Comment:
use pyproject.toml to define skip rule.
see official document's [this](https://github.com/codespell-project/codespell#using-a-config-file) section.

that's because skip will not be applied just running `codespell` ourselves.